src/precompile/modexp.rs

```diff
@@ -17,6 +18,9 @@ pub const BERNOULLI_LEN_LIMIT: U256 = U256::from_limbs([32, 0, 0, 0]);
 /// The MODEXP precompile with BERNOULLI length limit rule.
 pub const BERNOULLI: Precompile = Precompile::new(PrecompileId::ModExp, ADDRESS, bernoulli_run);
 
+/// The Galileo MODEXP precompile.
+pub const GALILEO: Precompile = Precompile::new(PrecompileId::ModExp, ADDRESS, galileo_run);
```

Comment:
Hmm can we do this?

```suggestion
pub const GALILEO: Precompile = Precompile::new(PrecompileId::ModExp, ADDRESS, modexp::osaka_run);
```

Reply:
yes, we can, I want to keep the same format with `bernoulli_run` at first. but seems don't need it.

Reply:
changed [8f5ba7d](https://github.com/scroll-tech/scroll-revm/pull/63/commits/8f5ba7d06b92745385c68db3fb2baf94efdfbe7a)